docker-compose-ci.yml

```diff
@@ -39,3 +40,7 @@ services:
     volumes:
       # only for running tests
       - ./test:/test
+
+  grafana:
+    ports:
+      - "3000:3000"
```

Comment:
Should this map to env variable grafana port instead of using constants ?

Reply:
- "${GRAFANA_PORT}:3000"

Reply:
same as above, I don't think these ports don't need to be opened at all, since `MonitorTest.py` is inside the network and uses the internal port (which is 3000). I deleted this section as CI checks pass